PeakLocalisations/Localisation/App Store/Metadata.xcstrings

```diff
@@ -51,6 +63,12 @@
             "state" : "translated",
             "value" : "Peak, widgets, fitness, health, goals, resolutions, habits, steps, workouts, weight tracker, weight tracking, pedometer"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "Peak, widgets, fitness, salud, metas, resoluciones, hábitos, pasos, entrenamiento, peso, seguimiento de pesos, pedómetro"
```

Comment:
seguimiento de peso (singular)
Also, I suggest writing podómetro instead of pedómetro, both are technically correct but I don't think the latter is used much at all